src/httpserver.cpp

```diff
@@ -464,21 +476,23 @@ void StopHTTPServer()
         delete workQueue;
         workQueue = nullptr;
     }
+    // Disable reading on current connections
+    for (evhttp_connection* conn : g_http_connections) {
```

Comment:
wouldn't there be unprotected concurrent acceess to `g_http_connections` from multiple threads here? (the HTTP event thread as well as the thread in which `StopHTTPServer` is called)